Makefile

```diff
@@ -147,6 +149,9 @@ $(MOCKGEN): $(TOOLS_DIR)/go.mod # Build mockgen from tools folder.
 $(RELEASE_NOTES) : $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR) && go build -tags tools -o $(BIN_DIR)/release-notes sigs.k8s.io/cluster-api/hack/tools/release
 
+$(GO_APIDIFF): $(TOOLS_DIR)/go.mod
+	cd $(TOOLS_DIR) && go build -tags=tools -o $(GO_APIDIFF_BIN) github.com/joelanford/go-apidiff
```

Comment:
This would change with https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/619

Reply:
I'll /hold until #619 merges

/hold